### PR TITLE
Weekly digest

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request is a weekly digest update.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,67 @@ description: |
...
+  date: May 23, 2019
+  link: https://github.com/magento/devdocs/pull/4588
+- description: Added the `isExpandable` and `suggestionDelay` Quick Search widget
+    [options](https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_quickSearch.html)
+    and fixed links to source code.
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: May 23, 2019
+  link: https://github.com/magento/devdocs/pull/4590
+- description: 'Reformatted YAML samples in the [Configure Environments: Services
```
Comment (lorikrell) [inline _data/whats-new.yml]: Is this a random single quote around this sentence ' ', also period?

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,67 @@ description: |
...
+    section of the Cloud documentation](https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services.html)'
+  versions: 2.x
+  type: Technical changes
+  date: May 22, 2019
+  link: https://github.com/magento/devdocs/pull/4574
+- description: Added a [new tutorial](https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/dynamic-row-system-config.html)
+    that shows you how to add a dynamic rows system configuration
```
Comment (lorikrell) [inline _data/whats-new.yml]: Just a period

```suggestion
    that shows you how to add a dynamic rows system configuration.
```

File: _data/whats-new.yml (hunk 1 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: The quote is added by the YAML generator. It is added when the string contains special symbols. `YAML` or `:`could be special in this case I guess.
The period has to be added to the original PR #4574.

File: _data/whats-new.yml (hunk 2 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: The period has to be added into `whatsnew` at the original PR #4528.

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**